purchases/src/test/java/com/revenuecat/purchases/utils/PriceExtensionsPricePerMonthTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.revenuecat.purchases.utils
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.models.Period
+import com.revenuecat.purchases.models.Price
+import junitparams.JUnitParamsRunner
+import junitparams.Parameters
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import org.junit.runner.RunWith
+
+@RunWith(JUnitParamsRunner::class)
+@Category(AndroidJUnit4::class)
+class PriceExtensionsPricePerMonthTest: PriceExtensionsPricePerPeriodTest() {
+    @Test
+    @Parameters(
```

Comment:
Parameterized tests FTW. It allows us to write repeated tests without duplicating code, and they work great with the IDE:

![Screenshot 2023-11-01 at 09 46 20](https://github.com/RevenueCat/purchases-android/assets/685609/6da59914-4234-4a46-b045-b4682584f533)

Reply:
Nice! yeah, I once tried to update to Juni5 to use them, but they were not compatible with Robolectric, which we use in a bunch of our tests... But adding a library like this for tests makes sense 👍